org.virtualxt.VirtualXT.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
this file should be submitted upstream. A Linux GUI app without a desktop file: it's a bug.

Reply:
Yes. It will be included in the next stable release.